### PR TITLE
Add checkbox and clear functions

### PR DESCRIPTION
**Pull request by dvcorreia1984**
In this pull request the following was done: 

1. New module was created for checkbox and clear functions
2. Object status was updated uppon click. 
3. A clear all function was added.

### CODE REVIEW

File: src/display.js [hunk 1]
```diff
@@ -45,25 +45,19 @@ todoList.id = 'todo';
...
+  const completedTasks = tasks.filter((task) => task.completed === true);
+  completedTasks.forEach((task) => {
+    const taskIndex = tasks.indexOf(task);
+    tasks.splice(taskIndex, 1);
   });
-}
+  localStorage.setItem('tasks', JSON.stringify(tasks));
+  // eslint-disable-next-line no-undef
+  renderTodoList();
+});
```
Comment (cynthiainga) [inline src/display.js]: - [ ] As mentioned in the project requirements you should create a new file for this functionality and import it as a module, kind;y do that 🙃 
- [ ] Also when a task is checked as complete it should be updated in the local storage completed: true but for you, it remains false even the box is checked, kindly fix that 🙃 😉 

Screenshot 2023-07-22 at 22 07 59

- [ ] Also kindly do not disable linters, fixing linters help us to write a clean, clear, and strong codebase 💪🏾

File: src/style.css [hunk 2]
```diff
@@ -96,15 +96,20 @@ div#addList {
...
   align-items: center;
-  padding: 20px;
-  font-size: medium;
-  background-color: #f6f6f6;
+  color: black;
+  font-weight: 600;
+}
+
+button#clear :hover {
+  /* stylelint-disable-next-line declaration-colon-space-after */
```
Comment (cynthiainga) [inline src/style.css]: - [ ] Kindly do not disable linters, fixing linters help us to write a clean, clear, and strong codebase 💪🏾

**Review by cynthiainga — CHANGES_REQUESTED** [review]
Hi @dvcorreia1984, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights

- [ ] Hhas event listener on the checkbox (`change`) ✔️ 
- [ ] The `Clear all completed` button works ✔️ 
- [ ] Correct GitHub flow ✔️ 

## Required Changes ♻️

- [ ] Please kindly check the comments under the review 👇🏾 👇🏾 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code._


Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/interactivity.js [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+        const description = listItem.querySelector('.description');
+        description.style.textDecoration = 'none';
+        description.style.color = '#333333';
+        task.completed = false;
+      }
+
+      // eslint-disable-next-line no-undef
+      localStorage.setItem('tasks', JSON.stringify(tasks));
+    });
+  });
```
Comment (anagudelogu) [inline src/interactivity.js]: You did a great job working on the previous reviewer's requirements! 💯 However, there's still room to improve:

- [ ] Please avoid disabling linter rules as they are very useful to keep a consistent code format and ensure you're using best practices! 🤓

File: src/style.css (hunk 2 above)
Comment (anagudelogu) [reply src/style.css]: - [ ] According to the previous reviewer, please avoid disabling linter rules as they are very useful to keep a consistent code format and ensure you're using best practices! 🤓

**Review by anagudelogu — CHANGES_REQUESTED** [review]
# STATUS: CHANGES REQUIRED ♻️ 🚧 

Hi @dvcorreia1984 👋🏼 
I'm Andres.

You've done a great job implementing some of the requested changes for this milestone 🚀, but there are still some which haven't been addressed yet. You're getting closer, keep up the good work! 🤖 🎆 



 ## Highlights
✅ Great use of GitHub flow. 💯 
✅ Your project is documented professionally. 📜 
✅ Great UI design. 💻 
✅ Descriptive PR title, summary and commit messages. 💬 


 ## Required Changes ♻️
_Check the comments under the review._

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by codepantha — APPROVED** [review]
# STATUS: APPROVED 🟢 ✔️ 

## Hi @dvcorreia1984 👋, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights 🎯 
✔️ Tasks can be marked as completed 📌 
✔️ "Clear all completed" functionality implemented 📌 
✔️ No linter errors

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**